pkg/cmd/set_replication.go

```diff
@@ -239,14 +240,20 @@ func (o *FinalizeOptions) SetReplication() error {
 		}
 	} else {
 		// if destPVC is empty, destination PVC name will be generated from source PVC
-		destOpts := DestinationOptions{
-			DestPVC: o.destPVC,
+		destOpts := ResourceOptions{
+			PVC: o.destPVC,
 		}
 		repOpts := &SetupReplicationOptions{
-			RepOpts:      o.RepOpts,
-			DestOpts:     destOpts,
-			SourcePVC:    srcPVC.Name,
-			StorageClass: o.destStorageClass,
+			RepOpts: o.RepOpts,
+			Dest: DestinationOptions{
+				destOpts,
+			},
```

Comment:
nit: Inline the destOpts (L243,244) like below with SourceOptions, for consistency, (or define sourceOpts above, like destOpts) but I'm fine with it as/is

Reply:
Sure